deploy/crds/integreatly.org_installations_crd.yaml

```diff
@@ -75,6 +75,7 @@ spec:
                 code after modifying this file Add custom validation using kubebuilder
                 tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html'
           required:
+          - lastError
```

Comment:
CASE In POINT: Here's a good example of someone having forgotten to run `make code/gen` or `operator-sdk generate openapi` before merging changes back to master.